pyuvdata/uvcal/initializers.py

```diff
@@ -74,18 +78,24 @@ def new_uvcal(
         Array of Jones polarization integers. If a string, options are 'linear' or
         'circular' (which will be converted to the appropriate Jones integers as
         length-4 arrays).
-    delay_array : ndarray of float, optional
-        Array of delays in seconds. If cal_type is not provided, it will be set to
-        'delay' if delay_array is provided.
     cal_type : str, optional
         Calibration type. Options are 'delay', 'gain'. Forced to be 'gain' if
         ``freq_array`` is given, and by *default* set to 'delay' if not.
     integration_time : float or ndarray of float, optional
-        Integration time in seconds. If not provided, it will be set to the minimum
-        time separation in time_array for all times.
+        Integration time in seconds. If not provided, it will be derived from the
+        time_array, as the difference between successive times (with the last time-diff
+        appended). If not provided and the number of unique times is one, then
+        a warning will be raised and the integration time set to 1 second.
+        If a float is provided, it will be used for all integrations.
+        If an ndarray is provided, it must have the same shape as time_array (or
+        unique_times, if that is what is provided).
```

Comment:
I don't think that `unique_times` is an input to this method.

Reply:
Good catch. Fixed.